src/cache.js

```diff
@@ -1,18 +1,24 @@
 import { shouldInvokePromise, calculateExpiry } from './utils';
 
-const createCache = (promiseFunc, options = {}) => {
-  const cache = {};
+const cache = {};
```

Comment:
This is moved outside of the `createCache` method as otherwise each invocation will have it's own cache.

Reply:
As discussed, we'll leave this as global for now and see how it plays out.